cms/envs/common.py

```diff
@@ -2771,3 +2774,14 @@
 #### django-simple-history##
 # disable indexing on date field its coming django-simple-history.
 SIMPLE_HISTORY_DATE_INDEX = False
+
+REST_FRAMEWORK['DEFAULT_SCHEMA_CLASS'] = 'drf_spectacular.openapi.AutoSchema'
```

Comment:
Needs explanatory comments.
`# Enables drf-spectacular autogenerated swagger docs. This does not remove support for legacy swagger docs that live in /api-docs .`